engine/gui/src/gui.cpp

```diff
@@ -4409,7 +4409,7 @@ Result DeleteDynamicTexture(HScene scene, const dmhash_t texture_hash)
         lua_rawgeti(L, LUA_REGISTRYINDEX, script->m_InstanceReference);
         dmScript::SetInstance(L);
 
-        ret = dmScript::PCall(L, 0, LUA_MULTRET);
+        ret = dmScript::PCall(L, 0, 0);
```

Comment:
Looks good.
Are there any other places we have the similar code in?
E.g. script_module.cpp:216 ?

Reply:
That's true, in ReloadModule. But at least we're throwing away the values there.